<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditAdminPassDialog</class>
 <widget class="QWidget" name="EditAdminPassDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>366</width>
    <height>268</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>验证管理员密码</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>241</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;h1 style=&quot; margin-top:18px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:xx-large; font-weight:600;&quot;&gt;修改管理员密码&lt;/span&gt;&lt;/h1&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>91</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>确认新密码：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="newPass2Edit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>170</y>
     <width>191</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="okBtn">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>220</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelBtn">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>220</y>
     <width>91</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>72</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>新密码：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="newPassEdit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>130</y>
     <width>191</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="oldPassEdit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>90</y>
     <width>191</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>72</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>旧密码：</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
